### PR TITLE
[NFC] Fix var delcation in doc blocks in the Civi folder

### PR DESCRIPTION
**Pull request by seamuslee001**
ping @eileenmcnaughton @totten

### CODE REVIEW

File: Civi/API/Kernel.php [hunk 1]
```diff
@@ -28,7 +28,8 @@ class Kernel {
   protected $dispatcher;
 
   /**
-   * @var array<ProviderInterface>
+   * @var array
```
Comment (colemanw) [inline Civi/API/Kernel.php]:
```suggestion
   * @var ProviderInterface[]
```

File: Civi/API/Provider/ReflectionProvider.php [hunk 2]
```diff
@@ -41,7 +41,8 @@ public static function getSubscribedEvents() {
   private $apiKernel;
 
   /**
-   * @var array (string $entityName => array(string $actionName))
+   * @var array\
```
Comment (colemanw) [inline Civi/API/Provider/ReflectionProvider.php]:
```suggestion
   * @var array[]
```

File: Civi/API/Subscriber/DynamicFKAuthorization.php [hunk 3]
```diff
@@ -50,17 +50,20 @@ public static function getSubscribedEvents() {
...
   /**
-   * @var string, the entity for which we want to manage permissions
+   * @var string
+   *  the entity for which we want to manage permissions
    */
   protected $entityName;
 
   /**
-   * @var array <string> the actions for which we want to manage permissions
+   * @var array
```
Comment (colemanw) [inline Civi/API/Subscriber/DynamicFKAuthorization.php]:
```suggestion
   * @var string[]
```

File: Civi/API/Subscriber/WrapperAdapter.php [hunk 4]
```diff
@@ -32,7 +32,8 @@ public static function getSubscribedEvents() {
   }
 
   /**
-   * @var array(\API_Wrapper)
+   * @var array
```
Comment (colemanw) [inline Civi/API/Subscriber/WrapperAdapter.php]:
```suggestion
   * @var \API_Wrapper[]
```

File: Civi/Core/DAO/Event/PostDelete.php [hunk 5]
```diff
@@ -18,12 +18,13 @@
 class PostDelete extends \Symfony\Component\EventDispatcher\Event {
 
   /**
-   * @var DAO Object
+   * @var DAOObject
```
Comment (colemanw) [inline Civi/Core/DAO/Event/PostDelete.php]:
```suggestion
   * @var CRM_Core_DAO
```

File: Civi/Core/DAO/Event/PostUpdate.php [hunk 6]
```diff
@@ -18,7 +18,7 @@
 class PostUpdate extends \Symfony\Component\EventDispatcher\Event {
 
   /**
-   * @var DAO Object
+   * @var DAOObject
```
Comment (colemanw) [inline Civi/Core/DAO/Event/PostUpdate.php]:
```suggestion
   * @var \CRM_Core_DAO
```

File: Civi/Core/DAO/Event/PreDelete.php [hunk 7]
```diff
@@ -18,7 +18,7 @@
 class PreDelete extends \Symfony\Component\EventDispatcher\Event {
 
   /**
-   * @var DAO Object
+   * @var DAOObject
```
Comment (colemanw) [inline Civi/Core/DAO/Event/PreDelete.php]:
```suggestion
   * @var \CRM_Core_DAO
```

**Comment by colemanw** [comment]
I think this is superceded by #16379?

**Comment by seamuslee001** [comment]
closing in favour of #16379

**Pull request closed without merging**